### PR TITLE
feat: streamline, prep for vid walkthrough

### PR DESCRIPTION
**Pull request by chadoh**
The biggest cleanup here is to consolidate the content into a couple larger chapters, rather than many small chapters. Read individual commits for more info.

### CODE REVIEW

File: src/counter/intro.md [hunk 1]
```diff
@@ -1,5 +1,175 @@
...
+| `-> i8`    | The return type of the method. A signed 8-bit integer, matching `val`.
+| `self.val` | This accesses the field `val`. You will also notice that there is no `return` statement. This is because in Rust everything is an expression, and a function returns the value of the last expression, as long as there is no semicolon after it.
+
+Go ahead and play with this a bit. Do things that don't make sense, and your well-configured editor will yell at you. Looking at the errors will teach you about how Rust works. You can try each of these things, changing it back to how it was after each:
+
+* Change `-> i8` to `-> i16`
+* Change `-> i8` to `-> ()`, an empty [tuple](https://doc.rust-lang.org/stable/book/ch03-02-data-types.html#the-tuple-type)
+* Remove `-> i18` – note that you get the same error message as when you changed it to an empty tuple! Empty tuple is one of the closest things Rust has no a nothing/`null` type.
```
Comment (willemneal) [inline src/counter/intro.md]:
```suggestion
* Remove `-> i18` – note that you get the same error message as when you changed it to an empty tuple! Empty tuple is one of the closest things Rust has to a `void` type.
```

File: src/counter/updating.md [hunk 2]
```diff
@@ -96,3 +96,33 @@ near dev-deploy $(raen build --release -q)
...
+
+You may have heard that blockchains are _immutable_, that they provide permanent ledgers of all on-chain activity.
+
+How does this fit with the fact that you just upgraded a smart contract, and it was actually super easy? It's even called a _contract_—doesn't that imply that it should be hard to change? If it's so easy for a contract owner to alter the contract, how can anyone using it trust they won't be scammed?
+
+Great questions!
+
+On other blockchains, it is in fact very difficult to upgrade a contract. They work around this with proxy contracts and other patterns.
+
+But NEAR has that handy account system, where each account has multiple keys. We already learned about _Function Call Access Keys_ two units ago; that's what makes it safe to have a web-based wallet, with web apps that store a private key in local storage.
```
Comment (willemneal) [inline src/counter/updating.md]:
```suggestion
But NEAR's handy account system, where each account can have multiple keys. We already learned about _Function Call Access Keys_ two units ago; that's what makes it safe to have a web-based wallet, with web apps that store a restricted private key in a browser's local storage.
```

Comment (chadoh) [reply src/counter/updating.md]: Thanks; I incorporated this changes slightly differently

File: src/counter/subaccounts.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+export ACCOUNT=your-account.testnet
+```
+
+Ok, now you can create a subaccount:
+
+```bash
+near create-account counter.$ACCOUNT --masterAccount $ACCOUNT --initialBalance 2
+```
+
+This makes a call to the blockchain as `your-account.testnet` (that's the `masterAccount` part), creating a subaccount, `counter.your-account.testnet`. It sends 2 NEAR tokens from `your-account.testnet` to the new account.
```
Comment (willemneal) [inline src/counter/subaccounts.md:54]: I think we should increase this to at least 6 N. It's the minimum an account needs in order to deploy, according to astro.  Though we should test this!

Comment (chadoh) [reply src/counter/subaccounts.md:54]: I've tested it with Counter an 2 was enough. The default for new accounts on testnet these days is only 10N! Blowing 6 of it on a `counter.` subaccount seems unfrugal.

File: src/counter/subaccounts.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+```bash
+near state counter.$ACCOUNT
+```
+
+And you can go enter `counter.your-account.testnet` into RAEN Admin. It should look the same as the old `dev-1234-1234` contract. But such a better name!
+
+## Summary
+
+Over this whole Counter chapter, we saw our first Rust code: a simple NEAR smart contract that implements a counter. Anyone can call `get_num` to see the current value, and anyone can change the number, too! We learned about Cargo, Rust's package manager, in order to fix a bug. We saw that updating contracts is easy thanks to Full Access keys, and that we only need to refresh RAEN Admin to see the latest contract changes.
```
Comment (willemneal) [inline src/counter/subaccounts.md:97]:
```suggestion
In this chapter we saw our first Rust code: a simple NEAR smart contract that implements a counter. Anyone can call `get_num` to see the current value, and anyone can change the number, too! We learned about Cargo, Rust's package manager, in order to fix a bug. We saw that updating contracts is easy thanks to _Full Access_ keys, and that we only need to refresh RAEN Admin to see the latest contract changes.
```

File: src/counter/subaccounts.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+In this unit we experienced more of NEAR's account name system, adding a Full Access key to our local machines by using `near login`. This allowed us to create a subaccount, which also added a Full Access key for that new subaccount. And we deployed the Counter contract to that new account.
+
+Overall, this Counter was a good way to learn some Rust and NEAR basics, but pretty silly as a real contract. Next, let's take a look at an example that does something more useful, storing different information for every user who calls the contract.
```
Comment (willemneal) [inline src/counter/subaccounts.md:101]:
```suggestion
Overall, a Counter was a good way to learn some Rust and NEAR basics, but pretty silly as a real contract. Next, let's take a look at an example that does something more useful, storing different information for every user who calls the contract.
```

**Pull request merged**